filters/include/pcl/filters/impl/conditional_removal.hpp

```diff
@@ -90,11 +90,8 @@ pcl::FieldComparison<PointT>::FieldComparison (
 template <typename PointT>
 pcl::FieldComparison<PointT>::~FieldComparison () 
 {
-  if (point_data_ != nullptr)
-  {
-    delete point_data_;
-    point_data_ = nullptr;
-  }
+  delete point_data_;
+  point_data_ = nullptr;
```

Comment:
As you deleted `point_data_` within the destructor there is no need to set it to `nullptr` after this.

Reply:
Yeah, I didn't check where I'm making the edits 😄  Will remove it soon